libs/ui-lib/src/lib/custom-time-picker/custom-time-picker.spec.tsx

```diff
@@ -20,13 +20,6 @@ describe('CustomTimePicker', () => {
     expect(baseElement).toBeTruthy();
   });
 
-  it('should render successfully', () => {
```

Comment:
I see you've taken out a duplicate unit test, so nice catch!
However, please state that you had refactored in the commit as well (and also add in the `refactor` label on your PR) 😄

Reply:
I'll be cleaning up my commits as I rebase